src/zenml/config/constants.py

```diff
@@ -0,0 +1,17 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""ZenML settings constants."""
+
+DOCKER_SETTINGS_KEY = "docker"
```

Comment:
Nice, it would probably make sense to move other constants to submodules as well, the current constants.py is quite poluted on our ZenServer branch atm @AlexejPenner